Gemfile

```
@@ -174,12 +174,16 @@ end
 group :production do
   # sidekiq enterprise requires a license key to download but is only required in production.
   # for local dev environments, regular sidekiq works fine
-  unless (Bundler::Settings.new['enterprise.contribsys.com'].nil? ||
-          Bundler::Settings.new['enterprise.contribsys.com']&.empty?) &&
+  unless (Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com'].nil? ||
+          Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com']&.empty?) &&
          ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', '').empty?
     source 'https://enterprise.contribsys.com/' do
       gem 'sidekiq-ent'
       gem 'sidekiq-pro'
     end
+  else
+    puts "No credentials found to install Sidekiq Enterprise. This is fine for local development but you may not check in this Gemfile.lock."
```

Just a little more verbosity:
```suggestion
    puts "No credentials found to install Sidekiq Enterprise. This is fine for local development but you may not check in this Gemfile.lock with the Sidekiq gems removed"
```

---

Took your advice